resources/google-music-electron-small.svg

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Can we rename this file to `logo.svg`? That would feel more inline with `icon.svg` =/